packages/react/src/common/React.types.ts

```diff
@@ -0,0 +1,6 @@
+import type { ClassAttributes, Key, ReactNode } from 'react';
+
+export interface IReactProps<T> extends ClassAttributes<T> {
+  children?: ReactNode | undefined;
```

Comment:
can we just mirror the deprecated/remove interface and document it ? 

using `ClassAttributes`:
- complicates the inheritance chain
- this interface may change as well in the future which would force us to do this gymnastic again

```ts
export interface IReactProps<T> {
       children?: ReactNode | undefined;
        key?: Key | undefined;
        ref?: LegacyRef<T> | undefined;
}
```

Reply:
Agree.